src/main/java/se/sundsvall/chat/integration/db/model/ConversationEntity.java

```diff
@@ -0,0 +1,184 @@
+package se.sundsvall.chat.integration.db.model;
+
+import jakarta.persistence.CollectionTable;
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.util.List;
+import java.util.Objects;
+import org.hibernate.annotations.UuidGenerator;
+
+@Entity
+@Table(name = "conversation")
+public class ConversationEntity {
+
+	@Id
+	@UuidGenerator
+	@Column(name = "id")
+	private String id;
+
+	@ElementCollection(fetch = FetchType.EAGER)
+	@CollectionTable(name = "conversation_participants", joinColumns = @JoinColumn(name = "conversation_id", foreignKey = @ForeignKey(name = "fk_participants_conversation_id")))
+	private List<ParticipantEntity> participants;
+
+	@Column(name = "municipality_id")
+	private String municipalityId;
+
+	@Column(name = "namespace")
+	private String namespace;
+
+	@Column(name = "channel_id")
+	private String channelId;
+
+	@OneToMany(fetch = FetchType.EAGER)
+	@CollectionTable(name = "conversation_meta_data", joinColumns = @JoinColumn(name = "conversation_id", foreignKey = @ForeignKey(name = "fk_meta_data_conversation_id")))
```

Comment:
Do you need to have `@CollectionTable` here? Isn't all this defined in MetaDataEntity?